receiver/hostmetricsreceiver/internal/scraper/processscraper/metadata.yaml

```diff
@@ -183,7 +183,7 @@ metrics:
 
   process.open_file_descriptors:
     enabled: false
-    description: Number of file descriptors in use by the process.
+    description: Number of file descriptors in use by the process. This metric is only available on Linux.
```

Comment:
Can you put it in `extended_documentation` field so it's not passed in the payload

Reply:
Sure! I didn't know this existed, nice :+1: